recipes/nslsii/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "nslsii" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "c409688657771291fb2c32e95045d3f44432402ebfb41dae5bc7188c1d34c075" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - appdirs
+    - bluesky
+    - bluesky-kafka  # [not win]
+    - caproto
+    - databroker
+    - ipython
+    - ipywidgets
+    - matplotlib
+    - numpy
+    - ophyd
+    - psutil
+    - pyolog
+
+test:
+  imports:
+    - nslsii
+    - nslsii.setup
+    - nslsii.versioneer
```

Comment:
I don't see these files in https://github.com/NSLS-II/nslsii/tree/master/nslsii. Did you go to the `nslsii` dir before running the script to get imports?

Reply:
Ah, that's my bad I apologize.